js/sider/KoronaComponents/KoronaSchema.js

```diff
@@ -2,100 +2,260 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Radio, Checkbox } from 'nav-frontend-skjema';
-import { Hovedknapp } from 'nav-frontend-knapper';
-import { Sidetittel, Normaltekst } from 'nav-frontend-typografi';
+import Lenke from 'nav-frontend-lenker';
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import { Sidetittel, Element, Innholdstittel, Undertittel } from 'nav-frontend-typografi';
+import { Bjorn } from '@navikt/digisyfo-npm/lib/components/Hjelpeboble';
+import {
+    tilLesbarDatoUtenAarstall,
+} from '@navikt/digisyfo-npm';
+import Hjelpetekst from 'nav-frontend-hjelpetekst';
 import { arbeidsgiver as arbeidsgiverPt } from '../../propTypes';
+import { tilLesbarDatoMedArstall } from '../../utils/datoUtils';
+import KoronaDatePicker from './KoronaDatePicker';
+
 
 class KoronaSchema extends Component {
     constructor(props) {
         super(props);
         this.state = {
             periode: undefined,
             arbeidsgivere: props.arbeidsgivere,
+            valgtArbeidsgivere: [],
             arbeidssituasjon: undefined,
+            tidligereSyk: true,
```

Comment:
```suggestion
            tidligereSyk: false,
```
initial state false?